### DISCUSSION TITLE
ESP32S3 : can we have special builds (bluetooth) or a "mega" build

### DISCUSSION THREAD

**Discussion by g6094199** [arendst/Tasmota#19805] [Ideas]
hi,

got hand on a S3 with plenty of flash (N16R8) for a fraction of cost....

Now, that ESP with that much flash cost almost nothing it wonder if we could have a "mega" build with all the nice stuff at once. this could reduce complexity on choosing the "right" build.

alternatively it would be nice to have also a bluetooth build for the C/S series cause of their Bluetooth 5 (LE) support.

**Comment by Jason2866** [comment]
Maybe the provided unofficial builds statisfy your needs. https://tasmota.github.io/install/index_release.html
